spec/at-rules/for.md

```diff
@@ -53,6 +57,8 @@ To execute a `@for` rule `rule`:
 
   * Add a variable with `rule`'s `VariableName` as its name and `i` as its value
     to `scope`.
+
+    > Note that this variable will have the same unit that `from`.
```

Comment:
The spec relies on the fact that the `+` operator for numbers allows adding a unitless number (`direction` in this case) to any number (and preserves the original unit). This note makes this case explicit.

This is what is mostly missing in dart-sass currently, due to the fact that it implements `i` with a Dart `num` rather than making it a `SassNumber` during the iteration (and so this behavior is not automatically inherited from `SassNumber`).